lib/__tests__/ProjectLogsManager.test.ts

```diff
@@ -1,9 +1,19 @@
-// @ts-nocheck
-const { ProjectLogsManager } = require('../projects/ProjectLogsManager');
-const { getProjectConfig, ensureProjectExists } = require('../projects');
-const {
-  fetchProjectComponentsMetadata,
-} = require('@hubspot/local-dev-lib/api/projects');
+import { ProjectLogsManager } from '../projects/ProjectLogsManager';
+import { getProjectConfig, ensureProjectExists } from '../projects';
+import { fetchProjectComponentsMetadata } from '@hubspot/local-dev-lib/api/projects';
+
+const SUBCOMPONENT_TYPES = {
```

Comment:
The `enums` folder is not exported from LDL, so I added `SUBCOMPONENT_TYPES` to this test. Is that ok? Too brittle?

Reply:
I think eventually it would be nice to have a single source of truth for this, but seeing how we're updating the project schema anyways it might not be worth putting too much effort into cleaning this up now.